charts/external-secrets/values.yaml

```diff
@@ -1,2 +1,14 @@
 external-secrets:
   installCRDs: true
+
+secretStores:
+  enabled: true
+  url: ssh://git@github.com/weaveworks/secret-stores    # url for the git repository that contains the SecretStores
+  # tag: v1.0.0                                   # if you would like to download your secrets from specific tag
```

Comment:
i would use as a default a tag instead of main if we are providing a default experience .given that both profile version and kustomization would go hand by hand we should be able to manage this versions together